studio/src/lib/track.ts

```diff
@@ -8,11 +11,12 @@ declare global {
   }
 }
 
-const resetKoala = () => {
+const resetTracking = () => {
   if (typeof window === "undefined") {
     return;
   }
 
+  posthog.reset();
```

Comment:
This should be a noop as well when Posthog wasn't configured.

Reply:
After testing, this check isnt required even for track. Removed it altogether